frontend/src/scenes/web-analytics/tiles/WebAnalyticsTile.tsx

```diff
@@ -262,6 +263,16 @@ const BreakdownValueCell: QueryContextColumnComponent = (props) => {
                 )
             }
             break
+        case WebStatsBreakdown.DeviceType:
+            if (typeof value === 'string') {
+                return (
+                    <div className="flex items-center gap-1">
```

Comment:
Since the flags would render as text, they didn't required style adjustments that I could find, so I decided to add this here to keep things smooth on the line displays. Tell me what you guys think :)

Reply:
Maybe gap-2? It looks a bit clumped in your screenshot. In any case we should make it behave like the flags

Reply:
<img width="596" alt="image" src="https://github.com/user-attachments/assets/58ede75c-df6e-4421-87dd-a6546a62383d" />
Whichever you prefer, gap-2 indeed has some more breathing room.

Reply:
I like `gap-2` better, but it's your call :)